todo-app/src/components/DisplayTodo.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { markComplete } from "../actions/todoActions";
+ 
+const DisplayTodos = () => {
+  const allTodos = useSelector((state) => {
+    return Object.values(state.todos)
+  })
+
+  const dispatch = useDispatch();
+
+  const handleClick = (e) => {
+    dispatch(markComplete(e.currentTarget.value))
+  }
+
+  return (
+    <ul>
+      {allTodos.map((todo) => {
+        return todo.completed ?
+        <li key={todo.id} value={todo.id} style={{ textDecoration: "line-through" }} onClick={handleClick}>{todo.text}</li> :
```

Comment:
You can have a ternary here that sets the style based on the state of your todo completed statu, something like this


`style ={ textDecoration: todo.completed? "line-thought" : "" }`